lib/Dialect/Triton/IR/Ops.cpp

```diff
@@ -317,6 +317,8 @@ bool DotOp::verifyDims() {
 }
 
 //-- DotScaledOp --
+Value DotScaledOp::getA() { return getLhs(); }
```

Comment:
Where these methods are being invoked currently?

Reply:
e.g., `getA`? Maybe you can replace existing methods with new ones?

Reply:
[DotScaledOp uses LHS/RHS](https://github.com/triton-lang/triton/blob/bca378d1c631b6db22296adad6732df7ee947f4d/include/triton/Dialect/Triton/IR/TritonOps.td#L692-L693). I can change the definition to A/B. Might be better for consistency. I don't have a strong opinion here though. @lezcano

Reply:
no strong opinions either

Reply:
Okay changed to use A/B for consistency with 3d8f49e